src/xcode/ENA/ENA/Source/Scenes/FileScanner/FileScannerHUD.swift

```diff
@@ -0,0 +1,72 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+import UIKit
+
+class FileScannerHUD {
+
+	// MARK: - Init
+
+	init(
+		execute: @escaping () -> Void
```

Comment:
```suggestion
		show: @escaping () -> Void
```
Just an idea...

Reply:
execute is the code run while the hud is shown - i thin it better matches

Reply:
ah then i mixed there something up. Then it's fine for me.